crates/uv/tests/pip_compile.rs

```diff
@@ -1660,22 +1671,26 @@ fn conflicting_repeated_url_dependency() -> Result<()> {
     ----- stdout -----
 
     ----- stderr -----
-    error: Requirements contain conflicting URLs for package `werkzeug`:
-    - git+https://github.com/pallets/werkzeug.git@2.0.0
-    - git+https://github.com/pallets/werkzeug@3.0.0
+    error: Requirements contain conflicting URLs for package `anyio`:
+    - git+https://github.com/agronholm/anyio.git@4.3.0
+    - git+https://github.com/agronholm/anyio.git@4.0.0
```

Comment:
Is there a test case like this for a commit that does not match a tag? Do we display the commit that the tag resolves to in that case?

Reply:
I don't _think_ so. I could add that, though.

Reply:
I'd at least like a snapshot, we could consider improving the output in the future

Reply:
Sorry, yes, we do already have this (`incompatible_narrowed_url_dependency`).